src/openrct2-ui/windows/Guest.cpp

```diff
@@ -758,7 +758,9 @@ static void window_guest_overview_tab_paint(rct_window* w, rct_drawpixelinfo* dp
         return;
     }
 
-    if (peep->Is<Staff>() && peep->AssignedStaffType == StaffType::Entertainer)
+    auto* staff = peep->As<Staff>();
+    auto* guest = peep->As<Guest>();
+    if (staff != nullptr && staff->AssignedStaffType == StaffType::Entertainer)
```

Comment:
I'd move this statement ~10 lines further, to be just before the `if` that needs it.